### DISCUSSION TITLE
Clarity Blueprint

### DISCUSSION THREAD

**Discussion by Statusbit** [Blackymas/NSPanel_HA_Blueprint#409] [Ideas]
Is it possible to improve the clarity in the Blueprint? For me it is very long and confusing.

An idea:
![Screenshot_20230309-094259](https://user-images.githubusercontent.com/85311570/223969200-bbb0d322-7f7a-4afa-bd29-f09adb1d1b84.png)
naming Buttons

or group Button with a line (hr) or with blank lines ...

EDIT:
Maybe the info can also be hidden or listed only at the first element?
`ONLY light | switch | cover | input_boolean | automation | button | input_button | scene | person | script | binary_sensor | fan)`

**Comment by Blackymas** [comment]
With version 3.2 there are changes in the design of the Blueprint so that it becomes clearer

**Comment by edwardtfn** [comment]
- #1722
